.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,22 @@
-Please fill in the fields below to submit a pull request.  The more information
-that is provided, the better.
+<!--
+Before submitting a pull request:
+  * Please run linter and tests locally: "tox"
+  * Ensure your commits are signed-off-by: Use "commit --signoff"
+  * For more info, see docs/CONTRIBUTING.rst
 
-Fixes #<ISSUE NUMBER>
+Once commits are signed off and tested, describe the purpose and contents
+of the pull request below.
+-->
 
 **Description of the changes being introduced by the pull request**:
 
-**Please verify and check that the pull request fulfills the following
-requirements**:
 
+
+
+Fixes #<ISSUE NUMBER>
+
+
+<!-- Please check relevant options (you can remove the ones that do not apply) -->
```

Comment:
I wonder if we should just remove the items below. They are usually ignored, and even if not, I never found them particularly helpful, neither as reviewer nor as contributor.

Maybe we can include a reminder in the bullet list above, something like:

> ensure changes are tested and code docs are updated